static/js/main.js

```diff
@@ -4,6 +4,7 @@ window.shark = window.shark || {};
 
 window.shark.info = window.shark.info || {};
 window.shark.screenshots = window.shark.screenshots || {};
+window.shark.screenshot_counter = 0;
```

Comment:
This is a bit off.
I said store it in `window.shark.info`.

Reply:
Sorry about not following that direction closely.